inception-search-mtas/src/main/java/de/tudarmstadt/ukp/inception/search/index/mtas/MtasDocumentIndex.java

```diff
@@ -671,11 +680,17 @@ private void indexAllDocuments()
     {
         log.info("Indexing all annotation documents of project {}", project.getName());
 
+        int u = 0;
```

Comment:
This is really unreadable, especially with these variable names.

Reply:
OK.